server/src/test/integration/org/apache/cassandra/sidecar/testing/IntegrationTestModule.java

```diff
@@ -114,7 +120,15 @@ public SidecarConfiguration configuration()
                                                                       "password"))
                             .build();
         AccessControlConfiguration accessControlConfiguration = accessControlConfiguration();
+        CassandraInputValidationConfiguration inputValidationConfiguration
+        = new CassandraInputValidationConfigurationImpl(DEFAULT_FORBIDDEN_KEYSPACES,
+                                                        // some tests generate table folders with -
+                                                        "[a-zA-Z][a-zA-Z0-9_-]{0,47}",
```

Comment:
this is not correct. For the stream sstable from snapshot endpoint we will get a table from the path that has the tableId. We should remove tableId from the path to be able to correctly validate. Otherwise my suspicion is that you will always get unauthorized for the table, because it won't match the table name

Reply:
Good catch, will update this